server/src/models/io/routines.ts

```diff
@@ -0,0 +1,12 @@
+import { ApiResponse } from "./util";
+import { ExerciseRoutines } from "../db/types"
+
+export type RoutineDataResponse = Omit<ExerciseRoutines, "routine_id">;
```

Comment:
Add routine_id as a number after omitting, unless you aren't responding with the id (I think you are?)

`export type RoutineDataResponse = Omit<ExerciseRoutines, "routine_id"> & { routine_id: number };`